finat/fiat_elements.py

```diff
@@ -278,11 +298,6 @@ def __init__(self, cell, degree):
         super(RaviartThomas, self).__init__(FIAT.RaviartThomas(cell, degree))
 
 
-class DiscontinuousRaviartThomas(VectorFiatElement):
-    def __init__(self, cell, degree):
-        super(DiscontinuousRaviartThomas, self).__init__(FIAT.DiscontinuousRaviartThomas(cell, degree))
```

Comment:
Due to the removal of this, this pull request must be merged at the same as firedrakeproject/tsfc#147.